upgrade-2.x.sh

```diff
@@ -992,9 +992,17 @@ LOGFILE="/mnt/data/balenahup/$SCRIPTNAME.$(date +"%Y%m%d_%H%M%S").log"
 mkdir -p "$(dirname "$LOGFILE")"
 echo "================$SCRIPTNAME HEADER START====================" > "$LOGFILE"
 date >> "$LOGFILE"
+
 # redirect all logs to the logfile, but also stderr to console (proxy)
-exec > >(cat >> "$LOGFILE")
-exec 2> >(tee -a "$LOGFILE" >&2)
+outfifo=$(mktemp -u)
+errfifo=$(mktemp -u)
+mkfifo "${outfifo}" "${errfifo}"
```

Comment:
Very clever. I never thought to use a dry-run of mktemp to generate unique filenames for another command to create.